src/com/itacademy/eshop/simulations/ManagerSimulation.java

```diff
@@ -27,22 +29,35 @@ private void addThreeNewProducts() {
         /**
          * adds three new products to the shop. One of them should be a book, one should be a Laptop, and one should be a shirt.
          */
-        Product laptop = new Product("Laptop", 1000, Category.ELECTRONICS);
+        Product laptop = new Product("Laptop", 1000, Category.ELECTRONICS, State.WITH_DEFECTS);
+        Product book = new Product("Book", 56.8, Category.BOOKS, State.NO_DEFECTS);
+        Product shirt = new Product("Shirt", 28, Category.CLOTHING, State.WITH_DEFECTS);
+
+        shop.addProduct(laptop);
+        shop.addProduct(book);
+        shop.addProduct(shirt);
     }
 
     private void findAndRemoveOneProduct() {
+                                                            // method returns Product, so we need to create it.
+        shop.removeProductByName("Laptop");         // method returns void, so no need to create Project object.
```

Comment:
avoid having comments which bring no value. for now maybe it is fine. But later you won't need to have them